lib/Optimizer/GraphOptimizer/PassManager.cpp

```diff
@@ -106,6 +118,10 @@ bool FunctionPassManager::runPrePass(const FunctionPass &P) {
       listContainsString(verifyBeforePassesOpt, P.getName())) {
     CHECK(F_->verify());
   }
+  if (printPassesOpt) {
+    LOG(INFO) << "Running Function Pass #" << globalPassCounter << ": "
```

Comment:
We probably want to print the name of the function (and graph) as well.

Reply:
Not sure what you mean by function and graph -- I added the name of the Function for now.